apps/site/components/ExtendableQueryTable/ExtendableQueryTable.tsx

```diff
@@ -0,0 +1,32 @@
+import { ReactNode } from "react"
+
+export type ExtendableQueryTableProps = {
+    children: ReactNode
+}
+
+const ExtendableQueryTable = ({ children }: ExtendableQueryTableProps) => {
+    return (
+        <table className="nx-simple-table">
+            <thead>
+                <tr>
+                    <th>Fragment</th>
+                    <th>Side</th>
+                    <th>Query operation</th>
+                    <th>Page</th>
+                    <th>Where is used</th>
+                    <th>Return</th>
+                    <th>Parameters</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    {
+                        children
+                    }
+                </tr>
+            </tbody>
+        </table>
+    )
+}
```

Comment:
This spacing is using 4, when it should be 2. Run prettier for this file pls

Reply:
done [here](https://github.com/vtex/faststore/pull/1942/commits/59f909d3f32fb15692f41e90dde311f9c74adc4c).